compat/src/test/scala/scala/jdk/CollectionConvertersTest.scala

```diff
@@ -85,6 +85,11 @@ class CollectionConvertersTest {
       assert((s: m.Map[String, String]) == p) // not eq
     }
 
+    {
+      val t: m.PriorityQueue[String] = it.to(m.PriorityQueue)
```

Comment:
I suggest that you move this code the `testTo` method in the `CollectionTest.scala` file. The simple fact that `.to(m.PriorityQueue)` does compile and does return a collection of type `m.PriorityQueue` is enough, IMHO, but you could also check that the content of the returned collection is correct as well.